shared/common-adapters/dropdown.js

```diff
@@ -84,7 +86,9 @@ class Dropdown<N: React.Node> extends React.Component<Props<N> & OverlayParentPr
             {this.props.items.map((i, idx) => (
               <ClickableBox
                 key={idx}
-                onClick={() => {
+                onClick={(evt: SyntheticEvent<>) => {
+                  evt.stopPropagation && evt.stopPropagation()
+                  evt.preventDefault && evt.preventDefault()
```

Comment:
@buoyad I'm having trouble getting this to actually prevent default. Any suggestions?